hack/svc/multi-stream.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import shutil
+import time
+import re
+import getopt
+import sys
+from subprocess import Popen
+
+
+class strims_ffmpeg:

```

Comment:
Lets get rid of this class and inline `hostAddress` to the main func. I typically avoid classes in simple scripts.